code/script.js

```diff
@@ -1,7 +1,32 @@
 // Variables that point to selected DOM elements
 const chat = document.getElementById('chat');
+const inputWrapper = document.getElementById('input-wrapper')
+const nameInput = document.getElementById('name-input')
+const nameForm = document.getElementById('name-form')
+
 
 // If you need any global variables that you can use across different functions, declare them here:
+const cursor = document.querySelectorAll(".cursor");
```

Comment:
I learned something new from this whole section, thank you for that and for being so creative!